docs/docs/legal/service-description/billing.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Pricing and billing of ZITADEL services
+sidebar_label: Billing
+custom_edit_url: null
+--- 
+
+This annex of the [Framework Agreement](../terms-of-service) describes the pricing and billing of our Services.
+
+Last revised: November 9, 2023
+
+## Pricing
+
+You can find pricing information on our [website](https://zitadel.com/pricing).
+
+### Enterprise pricing
+
+Customer and ZITADEL may agree on an individual per-customer pricing via an Enterprise Agreement.
+
+## Billing Metrics
+
+### Monthly quota
+
+Monthly quota means the available usage per measure for one billing period.
+The quota is reset to zero with the start of a new billing period.
+
+### Daily active user (DAU)
+
+Daily Active Users (DAU) are counted as users who authenticate or refresh their token during the given day.
+To calculate the monthly quota we take the sum of DAU over a given month.
```

Comment:
I am not sure about the word quota here. Would it not be better if we use "amount"